### PR TITLE
Add link to meta block tutorial

### PR DESCRIPTION
**Pull request by mkaz**
## Description

The documentation suggests that meta data be stored using blocks, this PR adds a link to the tutorial showing how to do so.

## Types of changes

Documentation.

### CODE REVIEW

File: docs/designers-developers/developers/backward-compatibility/meta-box.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Meta Boxes
 
-This is a brief document detailing how meta box support works in Gutenberg. With the superior developer and user experience of blocks, especially once block templates are available, **porting PHP meta boxes to blocks is highly encouraged!**
+This is a brief document detailing how meta box support works in Gutenberg. With the superior developer and user experience of blocks, especially once block templates are available, **porting PHP meta boxes to blocks is highly encouraged!** See the [Meta Block tutorial](/docs/designers-developers/developers/tutorials/metabox/meta-block-1-intro/) for how to store post meta data using blocks.
```
Comment (gziolo) [inline docs/designers-developers/developers/backward-compatibility/meta-box.md]: Url is broken when I click the link in view mode.

Comment (chrisvanpatten) [reply docs/designers-developers/developers/backward-compatibility/meta-box.md]: I think it needs to end with `.md`, not the trailing backslash

Comment (chrisvanpatten) [inline docs/designers-developers/developers/backward-compatibility/meta-box.md]:
```suggestion
This is a brief document detailing how meta box support works in Gutenberg. With the superior developer and user experience of blocks, especially once block templates are available, **porting PHP meta boxes to blocks is highly encouraged!** See the [Meta Block tutorial](/docs/designers-developers/developers/tutorials/metabox/meta-block-1-intro.md) for how to store post meta data using blocks.
```

**Review by chrisvanpatten — APPROVED** [review]
Just catching that `/ → .md` typo! Approving so it can be merged quickly once fixed.

**Pull request merged**